repository/src/test/java/com/iluwatar/repository/RepositoryTest.java

```diff
@@ -66,7 +65,7 @@ void setup() {
 
   @Test
   void testFindAll() {
-    var actuals = Lists.newArrayList(repository.findAll());
+    List<Person> actuals = (List<Person>) repository.findAll();
```

Comment:
```suggestion
    var actuals = (List<Person>) repository.findAll();
```

Reply:
You could add the method `List<Person> findAll` to `PersonRepository` and remove these casts.

Reply:
Made the changes according to comment.